jujugui/static/gui/src/app/components/charmbrowser/test-charmbrowser.js

```diff
@@ -43,14 +43,12 @@ describe('Charmbrowser', function() {
         }
       }};
     var changeState = sinon.stub();
-    var addService = sinon.stub();
     var charmstoreSearch = sinon.stub();
     var output = jsTestUtils.shallowRender(
       <juju.components.Charmbrowser
         appState={appState}
         changeState={changeState}
-        charmstoreSearch={charmstoreSearch}
-        addService={addService} />);
```

Comment:
You're correct in that `addService` is not currently being used, but when you go to make the 'Add to canvas' button work you'll want to use that method. Up to you when you want to add that back in :).